Library/Formula/pngcheck.rb

```diff
@@ -4,9 +4,10 @@ class Pngcheck < Formula
   homepage 'http://www.libpng.org/pub/png/apps/pngcheck.html'
   url 'https://downloads.sourceforge.net/project/png-mng/pngcheck/2.3.0/pngcheck-2.3.0.tar.gz'
   sha1 'e7f1535abbf2f809e036a9a43c759eeac5e39350'
+  revision 1
 
   def install
-    system 'make pngcheck'
-    bin.install 'pngcheck'
+    system 'make -f Makefile.unx ZINC=-I/usr/include ZLIB=/usr/lib/libz.dylib'
```

Comment:
Why do we need to hardcode the ZINC/ZLIB variables here? ZINC won't be correct on Xcode-only systems that don't have a `/usr/include`.